src/v2/models/AppState.js

```diff
@@ -185,6 +185,8 @@ export default Model.extend({
 
     if (identicalResponse && !isSameRefreshInterval) {
       this.set('dynamicRefreshInterval', this.getRefreshInterval(transformedResponse));
+    } else {
+      this.trigger('change:dynamicRefreshInterval');
```

Comment:
I think this shall be good enough?

```
if (identicalResponse) {
  this.set('dynamicRefreshInterval', this.getRefreshInterval(transformedResponse));
}

Reply:
Updated

Reply:
I meant instead of creating if/else we could just do a single `if` block.

Reply:
Updated